src/lib/components/event/event-details-full.svelte

```diff
@@ -41,7 +41,7 @@
       />
       {#each Object.entries(attributes) as [key, value] (key)}
         {#if attributeGrouping[activePill]?.includes(key)}
-          <EventDetailsRowExpanded {key} {value} class="w-full" />
+          <EventDetailsRowExpanded {key} {value} {attributes} class="w-full" />
```

Comment:
How can we improve that to only forward what we need (i.e the workflow id + run id only) ?

Reply:
Thanks @lminaudier! There's no good way to do this currently, we are planning on breaking apart the rows to be more specific to the type of data it shows. But that will be a much bigger refactor and we'd like to get this improvement out. I'll push up a PR that has these changes plus showing the link in the collapses view.